src/components/Header.style.js

```diff
@@ -0,0 +1,31 @@
+import styled from 'styled-components';
+import { Link } from 'react-router-dom';
+
+
+export const HeaderStyled = styled.header`
+  padding: 0 80px;
+  height: 100px;
+  display: flex;
+  justify-content: space-between;
+`;
+
+export const Nav = styled.nav`
+  display: flex;
+  align-self: center;
+`;
+
+export const NavLinkWrap = styled.div`
+  a {
```

Comment:
This should be a separate component like `export const NavLink = styled(Link)` IMO